runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/artifact/AbstractArtifactStagingService.java

```diff
@@ -50,6 +50,8 @@
 public abstract class AbstractArtifactStagingService extends ArtifactStagingServiceImplBase
     implements FnService {
 
+  public static final String NO_ARTIFACTS_STAGED_TOKEN = "__no_artifacts_staged__";
```

Comment:
Should this be put in a common location? (E.g. for use by boot.go and the Python staging.)

Reply:
For boot.go it's not required to check the token as the service will return the empty manifest. But for the py implementation it would be needed. I'm going to look at moving this to proto as follow-up.